src/Orchard/ContentManagement/DefaultContentManager.cs

```diff
@@ -78,7 +78,11 @@ public class DefaultContentManager : IContentManager {
         public ILogger Logger { get; set; }
 
         public IEnumerable<IContentHandler> Handlers {
-              get { return _handlers.Value; }
+            get { return _handlers.Value; }
+        }
+
+        public ITransactionManager TransactionManager {
```

Comment:
Why is it necessary, you can already resolve it.

Reply:
I need it in taxonomy extensions https://github.com/OrchardCMS/Orchard/pull/7861/files#diff-bb0c748b3dc7bb68f064a62685877941R35  There the only way I can get it is through an extra parameter on the extension. I avoided that getting the transaction from the content manager.